pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -34,7 +34,7 @@ import (
 	drapb "k8s.io/kubelet/pkg/apis/dra/v1alpha3"
 )
 
-const PluginClientTimeout = 10 * time.Second
+const PluginClientTimeout = 45 * time.Second
```

Comment:
the `PrepareResources` iterates over the plugins and each will now has increased timeout. Is there needed some paralleliation?